scripts/app/convenient-effects-controller.js

```diff
@@ -28,45 +28,54 @@ export default class ConvenientEffectsController {
    * @returns {Object} the data to pass to the template
    */
   get data() {
-    return {
-      folders: [
-        {
-          id: 'favorites',
-          name: 'Favorites',
-          effects: this._fetchFavorites(),
-        },
-        {
-          id: 'custom',
-          name: 'Custom',
-          effects: this._fetchUnfavoritedCustomEffects(),
-        },
-        {
-          id: 'conditions',
-          name: 'Conditions',
-          effects: this._fetchUnfavoritedConditions(),
-        },
-        {
-          id: 'spells',
-          name: 'Spells',
-          effects: this._fetchUnfavoritedSpells(),
-        },
-        {
-          id: 'class-features',
-          name: 'Class Features',
-          effects: this._fetchUnfavoritedClassFeatures(),
-        },
-        {
-          id: 'equipment',
-          name: 'Equipment',
-          effects: this._fetchUnfavoritedEquipment(),
-        },
-        {
-          id: 'other',
-          name: 'Other',
-          effects: this._fetchUnfavoritedOther(),
-        },
-      ],
-    };
+    const folders = [
+      {
+        id: 'favorites',
+        name: 'Favorites',
+        effects: this._fetchFavorites(),
+      },
+      {
+        id: 'custom',
+        name: 'Custom',
+        effects: this._fetchUnfavoritedCustomEffects(),
+      },
+      {
+        id: 'conditions',
+        name: 'Conditions',
+        effects: this._fetchUnfavoritedConditions(),
+      },
+      {
+        id: 'spells',
+        name: 'Spells',
+        effects: this._fetchUnfavoritedSpells(),
+      },
+      {
+        id: 'class-features',
+        name: 'Class Features',
+        effects: this._fetchUnfavoritedClassFeatures(),
+      },
+      {
+        id: 'equipment',
+        name: 'Equipment',
+        effects: this._fetchUnfavoritedEquipment(),
+      },
+      {
+        id: 'other',
+        name: 'Other',
+        effects: this._fetchUnfavoritedOther(),
+      },
+    ];
+    if (
+      this._foundryHelpers.moduleActiveVersionCheck('midi-qol', '11.0.10.99')
```

Comment:
Rather than have a whole separate folder for midi, let's use the `isViewable` flag on the `createActiveEffect` function.